### ISSUE TITLE
Book circulation lists and joint accounts. 

### ISSUE THREAD

**Issue by jkotin** [Princeton-CDH/mep-django#631]
## testing notes
Find book events for a few different shared accounts and check how the event displays on book circulation table and book card list — both names should be present; on the circulation view both should link to their individual account. On the card view, link will go to the first account.

----

The problem:

Leon Edel shares an account with his wife Bertha. They borrow James Farrell’s The Young Manhood of Studs Lonigan in 1936. The book shows up on the member pages of both Leon and Bertha, as it should. But on the book page, only Bertha shows up on the circulation list. We don't want the book to count twice in the circulation numbers.

There are five joint accounts so far insofar as I know:

Bertha & Leon Edel
Monroe Wheeler & Glenway Wescott 
Elisabeth de Roos & E. du Perron
Roger Caillois & Armand Petitjean 

@clmahoney indicates another Mr. and Mrs. William Deknatel, but I can't find it, and have asked her for a link.

There may be other undiscovered joint accounts. If we look back at the cards, we may find some Mr. and Mrs. together.

**Comment by rlskoeser** [comment]
@jkotin @gissoo how about this for circulation table and cards list?
![Screen Shot 2020-05-27 at 4 15 28 PM](https://user-images.githubusercontent.com/691231/83067867-610a5780-a035-11ea-9878-b4765680fbd3.png)
![Screen Shot 2020-05-27 at 4 15 37 PM](https://user-images.githubusercontent.com/691231/83067876-65367500-a035-11ea-9304-aec53d373c26.png)

The card still links to the first name because of the way I architected the card urls; not sure of a simple solution for that, but at least displaying both names is better than that.

**Comment by jkotin** [comment]
Looks good to me. How does it look with other cards -- is the spacing still grid-like? What's important to me is that both names are on the circulation lists too.

**Comment by rlskoeser** [comment]
card browse showing other cards:
![Screen Shot 2020-05-27 at 4 20 59 PM](https://user-images.githubusercontent.com/691231/83068281-205f0e00-a036-11ea-8cab-0a1260cd6e54.png)
circulation activity showing other rows:
![Screen Shot 2020-05-27 at 4 21 09 PM](https://user-images.githubusercontent.com/691231/83068290-2228d180-a036-11ea-9d95-9cf1b0ed1f12.png)
circulation activity mobile view:
![Screen Shot 2020-05-27 at 4 21 23 PM](https://user-images.githubusercontent.com/691231/83068303-26ed8580-a036-11ea-95e4-1abcd37a89c7.png)

**Comment by jkotin** [comment]
Looks perfect to me.  I'll wait on @gissoo to close.

**Comment by rlskoeser** [comment]
Don't close yet! I need to commit the code and have Cate review on the test site. But wanted to make sure you and @gissoo were both ok with the approach.

@gissoo any concerns or requests?

**Comment by gissoo** [comment]
@rlskoeser Thank you. It's fine, but the spacing between the date and the names does not seem to match the designs visually at least, on Zeplin it's 5px. It would be much better if it can be increased to match the designs (or I can make a separate issue for it). If the spacing between the two names are less 1px or less, it might be helpful to make them 2px.

**Comment by rlskoeser** [comment]
@gissoo you're talking about the spacing on the cards? I'll go ahead and adjust now.

**Comment by gissoo** [comment]
@rlskoeser Yes, on the cards browse page not the card image viewer. Thank you!!

**Comment by clmahoney** [comment]
- YES, members who have shared accounts now both show up in the circulation list if their card is at Princeton. I tried with the Edels and with Wescott/Wheeler: 
![Screen Shot 2020-05-28 at 12 10 53 PM](https://user-images.githubusercontent.com/16673934/83166534-104e3980-a0dd-11ea-84dc-e232afd8d989.png)
![Screen Shot 2020-05-28 at 12 11 38 PM](https://user-images.githubusercontent.com/16673934/83166537-117f6680-a0dd-11ea-8c25-8f1745d8289d.png)
![Screen Shot 2020-05-28 at 12 11 47 PM](https://user-images.githubusercontent.com/16673934/83166540-12b09380-a0dd-11ea-87b2-1c775a40951c.png)


BUT - I tried with de Roos and Peron too, but although de Roos shows up on the circulation list, her husband (Perron) doesn't. Their card doesn't show up since it's from the Ransome Center. But is there a reason why his name doesn't show up with hers in the circulation list? see here: 
![Screen Shot 2020-05-28 at 12 16 07 PM](https://user-images.githubusercontent.com/16673934/83166522-0d534900-a0dd-11ea-968d-644aa9b4f185.png)

**Comment by rlskoeser** [comment]
@clmahoney it doesn't look to me that de Roos and Perron are marked as a shared account in the database — here's a screenshot of an admin account search for "perron" on the test site:
![Screen Shot 2020-05-28 at 12 46 28 PM](https://user-images.githubusercontent.com/691231/83169663-5c02e200-a0e1-11ea-9544-6fa72329efdf.png)

**Comment by clmahoney** [comment]
@rlskoeser oh of course! They probably just aren't joined in the test site. But they are on the production site! This looks all good to me then.
